include/swift/AST/DiagnosticsSema.def

```diff
@@ -5954,8 +5954,7 @@ ERROR(concurrent_value_outside_source_file,none,
       "%kind0; use '@unchecked Sendable' for retroactive conformance",
       (const ValueDecl *))
 ERROR(concurrent_value_nonfinal_class,none,
-      "non-final class %0 cannot conform to 'Sendable'; "
-      "use '@unchecked Sendable'", (DeclName))
+      "non-final class %0 cannot conform to the 'Sendable' protocol", (DeclName))
```

Comment:
the "to the Sendable protocol" rewording because it seems that's how we spell it everywhere else

Reply:
Yep, looks good!